tracee-rules/engine/engine.go

```diff
@@ -334,3 +334,12 @@ func analyzeEventOrigin(event tracee.Event) string {
 		return EVENT_HOST_ORIGIN
 	}
 }
+
+// GetSelectedEvents returns the event selectors that are relevant to the currently loaded signatures
+func (engine *Engine) GetSelectedEvents() []types.SignatureEventSelector {
+	res := make([]types.SignatureEventSelector, 0)
+	for k := range engine.signaturesIndex {
+		res = append(res, k)
```

Comment:
If the answer to the above question is yes, then I suggest:
1. Only save the Name field of the SignatureEventSelector to get the event name
2. To avoid duplications (different signatures that ask for the same events), you can use a map instead